usmqe_tests/api/ceph/test_import.py

```diff
@@ -0,0 +1,100 @@
+"""
```

Comment:
I would rename this file to test_cluster_import.py so it is clear what is imported in the file and it is unified with gluster tests.